apps/ui/src/components/PickerToken.vue

```diff
@@ -143,9 +142,9 @@ watch(
       @click="handlePick(asset)"
     >
       <div class="flex items-center min-w-0 pr-2">
-        <UiBadgeNetwork :id="networkId">
+        <UiBadgeNetwork :chain-id="network">
           <UiStamp
-            :id="`${networkId}:${asset.contractAddress}`"
+            :id="`eip155:${network}:${asset.contractAddress}`"
```

Comment:
This icon seems to not always work

Before this pr, it looks like this

![Screenshot 2024-11-05 at 00 32 41](https://github.com/user-attachments/assets/ebd4e1c5-86ed-4f02-a4b4-fe9a948d0750)

After:

![Screenshot 2024-11-05 at 00 32 32](https://github.com/user-attachments/assets/7241c3ec-38e0-4411-ae59-664ed2cc7f29)